.github/main.workflow

```diff
@@ -6,7 +6,7 @@ workflow "build and test on branch" {
     "branch.lint.node.10",
     "branch.test.node.10",
     "branch.test.node.8",
-    # "branch.test.node.12"
+    "branch.test.node.12"
```

Comment:
I think @ayusharma detected an issue with Node12 but I don't remember what exactly.

Reply:
I will check this in the next week if there are any issues.